internal/node/node_repositories.bzl

```diff
@@ -19,710 +19,9 @@ See https://docs.bazel.build/versions/main/skylark/repository_rules.html
 """
```

Comment:
Why did we decide to keep this one? 

I would expect nodejs//:repositories to be referenced everywhere but I believe this was done to keep the changes minimal.

Reply:
yeah I extracted the primary rule from here, but to keep it non-breaking I had to keep the surrounding macro.

No users will install `rules_nodejs` workspace until we ask them to, which is a breaking change. So everything still has to be exposed in `build_bazel_rules_nodejs`, only private helper APIs can move.

Reply:
Got it!